src/composables/pollingDetails.js

```diff
@@ -1,35 +1,56 @@
 import { reactive, ref, onMounted, computed } from "vue";
 import { useStore } from "vuex";
 import { useRouter } from "vue-router";
-
 export const getAllPollsApi = () => {
+  const isLoading = ref(false);
   const store = useStore();
   const pageNo = ref(1);
   const limit = ref(4);
+  const pollsData = computed(() => {
```

Comment:
if you are not doing anything with pollsData variable, you can simply put it into the return object @ShivanshSinghnnovative